test/system/410-selinux.bats

```diff
@@ -75,4 +91,84 @@ function check_label() {
     run_podman rm -f myc
 }
 
+# Sharing context between two containers not in a pod
+# These tests were piggybacked in with #7902, but are not actually related
+@test "podman selinux: shared context in (some) namespaces" {
```

Comment:
Nice, there are a few more we might want to add around making sure a systemd based container gets the 
container_init_t and a kvm (kata) container gets container_kvm_t.